services/visual-studio-marketplace/visual-studio-marketplace-base.js

```diff
@@ -34,6 +35,16 @@ const extensionQuerySchema = Joi.object({
     .required(),
 }).required()
 
+const statisticSchema = Joi.object().keys({
+  install: Joi.number().default(0),
+  updateCount: Joi.number().default(0),
+  onpremDownloads: Joi.number().default(0),
+  averagerating: Joi.number()
+    .default(0)
+    .precision(2),
```

Comment:
Kinda lucked into this, as one of the live tests started failing today due to some rating changes in the marketplace. My current understanding is that this will result in the value for `averagerating` having a numeric value with a precision of 2 post the Joi validation (as opposed to failing validation if the value has more than 2 decimal places)

Reply:
Yea, I just confirmed that.

Is this for formatting? Would it be better to handle in `render()`?

Reply:
Yeah I suppose it is, good call. So this should be flipped back, and then in the services that use this field, they should format it as needed (in their `render` functions)?